templates/html/.filters/all.js

```diff
@@ -14,13 +14,12 @@ module.exports = ({ Nunjucks, Markdown, OpenAPISampler }) => {
     if (
       obj.type === 'object' ||
       obj.type === 'array' ||
-      obj.oneOf ||
-      obj.anyOf ||
-      obj.items ||
-      obj.additionalItems ||
```

Comment:
This should not be removed. It's a valid property. Check https://json-schema.org/latest/json-schema-validation.html#rfc.section.6.4.2.

That said, it's missing on the JS parser and we should add it there. I opened an issue: https://github.com/asyncapi/parser-js/issues/25.

Reply:
JS parser is now supporting `additionalItems`. You should just update the `asyncapi-parser` package to its latest version.